pallets/parachain-system/src/validate_block/tests.rs

```diff
@@ -134,6 +143,7 @@ fn validate_block_no_extra_extrinsics() {
 		witness,
 		validation_data,
 	} = build_block_with_witness(&client, vec![], parent_head.clone());
```

Comment:
Why isn't this function directly returning the `CompactProof`?

Reply:
:facepalm: done